UI/PlayerListWnd.cpp

```diff
@@ -765,6 +774,23 @@ std::set<int> PlayerListWnd::SelectedPlayerIDs() const {
     return retval;
 }
 
+std::set<int> PlayerListWnd::ReadyEmpiresIDs() const {
```

Comment:
comment explaining what this does and why?

Reply:
Added comment.